cmd/kpod/common.go

```diff
@@ -50,6 +50,8 @@ func getConfig(c *cli.Context) (*libkpod.Config, error) {
 	if c.Bool("debug") {
 		logrus.SetLevel(logrus.DebugLevel)
 	}
-
+	if c.GlobalIsSet("runtime") {
```

Comment:
If you add an option to common, you need to add documentation of the option to kpod.1.md, or make this a hidden option.